package-lock.json

```diff
@@ -0,0 +1,4420 @@
+{
```

Comment:
I think this repo uses `yarn` and `yarn.lock` instead. Could we commit that instead until we figure out the story for `package-lock.json`? It's also only for `npm >= 5` right?